bin/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :name
+  attr_reader :publish_date
+
+  def initialize(name, publish_date, on_spotify)
+    super(publish_date)
+    @name = name
```

Comment:
Please, can you check if there is any +name attribute in the UML diagram for this class?